docs/config.md

```diff
@@ -195,3 +195,35 @@ toc: false
 ## search
 
 Whether to enable [search](./search) on the project; defaults to false.
+
+## interpreters
+
+The interpreters option specifies additional languages for data loaders, indicating the extension and associated interpreter (see [routing](./loaders#routing) for details). The list of supported interpreters defaults to:
+
+```js run=false
+{
+  ".js": ["node", "--no-warnings=ExperimentalWarning"],
+  ".ts": ["tsx"],
+  ".sh": ["sh"],
+  ".exe": [""],
```

Comment:
Shouldn’t this be the empty array?

```suggestion
  ".exe": [],
```

Reply:
good catch! that's what is in the code, in fact.